### DISCUSSION TITLE
Mehrere Frequenzen gleichzeitig nutzen

### DISCUSSION THREAD

**Discussion by webstr4711** [Schrolli91/BOSWatch#531] [General]
Guten Abend, 

wie können mit einem Stick mehrere Frequenzen (in der Bandbreite die der Stick untgerstützt... z.b. 2 Mhz) beobachtet werden?
rtl-fm blockt ja den Stick.

Ich habe nun eine Windows-SDR-Software gesehen, mit der das möglich ist. Das heisst, technisch muss es grundsätzlich gehen.
Man kann dort verschiedene Freqzenzen einzeln anwählen und in virtuelle audio cable speisen.
Aber ich möchte das mit Linux/raspi machen.

Habt ihr da einen Ansatz?

Viele Grüße

**Comment by flothi** [comment]
Nach jetzigem Stand maximal mit zwei Sticks, also zwei Instanzen des Programms.
ggf kann Version 3 das von Hause aus besser, da bin ich aktuell aber gar nicht drin

**Comment by webstr4711** [comment]
danke für deine antwort.
wie kann man mehrere instanzen starten? einen parameter zur angabe einer (anderen) configdatei habe ich nicht gefunden - was doch zwingend voraussetzung wäre.

davon abgesehen möchte ich ja mehrere frequenzen mit einem stick nutzen.

rtl_fm unterstützt ja die angabe mehrerer frequenzen.

OPTIONS
       -f frequency_to_tune_to [Hz]
               use multiple -f for scanning, (requires squelch)
               ranges supported, -f 118M:137M:25k

**Comment by Schrolli91** [comment]
> rtl_fm unterstützt ja die angabe mehrerer frequenzen.

das haben wir bereits vor langer Zeit mal getestet - leider mit wenig Erfolg.
Das Frequenzhopping von rtl_fm ist zu langsam - dadurch bekommt MM dann Probleme mit der Auswertung und es gehen ggf. Pakete verloren.

**Comment by webstr4711** [comment]
okay, das ist schade, @Schrolli91 
wie funktioniert das von @flothi angesprochene starten zweier Instanzen? Man müsste ja eine andere config angeben..
